src/components/staking/PageValidators.vue

```diff
@@ -4,7 +4,7 @@
     :loading="$apollo.queries.validators.loading"
     :loaded="!$apollo.queries.validators.loading"
     :error="$apollo.queries.validators.error"
-    :data-empty="validators.length === 0"
+    :data-empty="validators && validators.length === 0"
```

Comment:
Was this causing a bug before? It defaults to an array, so it should be ok.

Really not a big deal, but it would mean a few less characters if we can do without it.